src/pages/Waiting/index.js

```diff
@@ -22,22 +21,15 @@ export default function WaitingRoom() {
   const roomIndex = id.split("-")[1];
   const roomName = `${channelIndex}-${roomIndex}`;
 
-  // 룸 아이디, 타이틀
   const roomDataInfo = location.state.roomData;
   const [roomTitle, roomDBID] = roomDataInfo.split("-");
```

Comment:
이 부분 카멜 케이스가 제대로 적용이 안된 것 같아서 이 부분은 제가 수정해서 Pr올릴께요!
```suggestion
  const [roomTitle, roomDbId] = roomDataInfo.split("-");
```